src/idfxt/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+set(SRC 
+  JSONDataInterface.cpp
+  JSONDataInterface.h
+  idd-full.h
```

Comment:
@refaqtor where is the code that generates the idd-full.h and idd-minimal-ordered.h files?  What is the content of the idd-minimal-ordered.h file?  Is that just object names or something?